src/v2/Apps/Fair/Components/__tests__/FairFollowedArtists.jest.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react"
+import { FairFollowedArtists } from "../FairFollowedArtists"
+import { useTracking } from "react-tracking"
+import { AnalyticsContext } from "v2/Artsy/Analytics/AnalyticsContext"
+import { mount } from "enzyme"
+import { OwnerType } from "@artsy/cohesion"
+
+jest.mock("react-tracking")
+jest.mock("v2/Components/Artwork/FillwidthItem", () => ({ onClick }) => (
+  <button onClick={onClick}>FillwidthItem</button>
+))
+
+const FAIR_FOLLOWED_ARTISTS_FIXTURE = {
+  id: "example-fair-id",
+  slug: "example-fair-slug",
+  followedArtistArtworks: {
+    edges: [
+      {
+        artwork: {
+          internalID: "example-artwork-internal-id",
+          slug: "example-artwork-slug",
+        },
+      },
+    ],
+  },
+}
+
+describe("FairFollowedArtists", () => {
+  let trackEvent
+
+  beforeEach(() => {
+    trackEvent = jest.fn()
+    ;(useTracking as jest.Mock).mockImplementation(() => {
+      return {
+        trackEvent,
+      }
+    })
+  })
+
+  const getWrapper = () => {
+    return mount(
+      <AnalyticsContext.Provider
+        value={{
+          contextPageOwnerId: "example-fair-id",
+          contextPageOwnerSlug: "example-fair-slug",
+          contextPageOwnerType: OwnerType.fair,
+        }}
+      >
+        <FairFollowedArtists fair={FAIR_FOLLOWED_ARTISTS_FIXTURE as any} />
+      </AnalyticsContext.Provider>
+    )
+  }
+
+  it("makes the appropriate analytics call when 'View all' is clicked", () => {
+    const wrapper = getWrapper()
+
+    const viewAll = wrapper
+      .find("a")
+      .findWhere(node => node.text() === "View all")
+      .first()
+
+    expect(trackEvent).not.toBeCalled()
+
+    viewAll.simulate("click")
+
+    expect(trackEvent).toHaveBeenCalledWith({
+      action: "clickedArtworkGroup",
+      context_module: "worksByArtistsYouFollowRail",
+      context_page_owner_id: "example-fair-id",
+      context_page_owner_slug: "example-fair-slug",
+      context_page_owner_type: "fair",
+      destination_page_owner_id: "example-fair-id",
+      destination_page_owner_slug: "example-fair-slug",
+      destination_page_owner_type: "fair",
+      type: "viewAll",
```

Comment:
I think this is correct? We're just linking to the filter view of the fair so it's fair => fair

Reply:
Seems right.